BlazarService/src/main/java/com/hubspot/blazar/listener/LaunchingRepositoryBuildVisitor.java

```diff
@@ -197,4 +210,24 @@ private boolean lastBuildSucceeded(Module module) {
 
     return moduleMap;
   }
+
+  private void enqueueModuleBuild(GitInfo branch, RepositoryBuild branchBuild, Module module) throws IOException, NonRetryableBuildException {
+    String sha = branchBuild.getCommitInfo().get().getCurrent().getId();
+    String folder = module.getFolder();
+    String configPath = folder + (folder.isEmpty() ? "" : "/") + ".blazar.yaml";
+    BuildConfig baseConfig = buildConfigUtils.getConfigAtRefOrDefault(branch, configPath, sha);
```

Comment:
I don't see any try/catch in the `visitLaunching` method. What will happen if this method throws and error?